### PR TITLE
API to send a email notification about a conflict in the client files.

### PR DESCRIPTION
**Pull request by mrakob**
Notification will be sent from seafile-client

Changes to be committed:
    new file:   seahub/api2/endpoints/send_notify_file_conflict.py
    modified:   seahub/api2/urls.py
    modified:   seahub/settings.py
    new file:   seahub/templates/send_email_notify_file_conflict.html

### CODE REVIEW

**Comment by freeplant** [comment]
Hi mrakob, what's the usage case of this API?

**Comment by shoeper** [comment]
@freeplant it should allow the client to send an email (to the user) in case there is a conflict.

**Comment by freeplant** [comment]
@mrakob do you call this API from desktop client? Have you changed the desktop client?

File: seahub/api2/endpoints/send_notify_file_conflict.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        path = request.data.get('path', '')
+
+        from_email = None  # use default from email
+
+        title = _(u'File conflict notification %s') % SITE_NAME
+        template = 'send_email_notify_file_conflict.html'
+
+        to_email = email
+
+        logger.warning('repo_id: %s repo_name: %s, path: %s', repo_id, repo_name, path)
```
Comment (shoeper) [inline seahub/api2/endpoints/send_notify_file_conflict.py]: What should this be good for?

**Comment by shoeper** [comment]
I like the idea but I would default the setting (IS_SEND_EMAIL_FILE_CONFLICT = True) to false. It should be an active decision for the administrator because it can lead to many emails (thousands of emails).

**Comment by freeplant** [comment]
I think the idea of letting the client call the server to send a conflict notification email is not good. If it is really need, it is better to let the server detects such conflicts and notify users.

**Pull request closed without merging**